app/controllers/comparisons/change_in_heater_since_last_year.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Comparisons
+  module ChangeInHeaterSinceLastYear
```

Comment:
Minor: wondering about changing the name here slightly. As its about either gas use (which is both hot water and heating) or storage heater use. The name seems to relate to the latter. Maybe `ChangeInHeatingSinceLastYear` and similar tweaks to prefix for partials?

Reply:
changed